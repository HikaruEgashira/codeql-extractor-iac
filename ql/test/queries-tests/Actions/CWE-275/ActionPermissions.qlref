security/Actions/CWE-275/ActionPermissions.ql