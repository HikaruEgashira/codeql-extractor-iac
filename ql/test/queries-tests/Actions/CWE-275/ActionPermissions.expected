| .github/workflows/nopermissions.yml:1:1:13:29 | name: "Sample" | Workflow does not contain permissions |
